### ISSUE TITLE
Documentation warnings and build issues occur when TMAP8 is built with SALAMANDER

### ISSUE THREAD

**Issue by cticenhour** [idaholab/TMAP8#322]
## Bug Description
When building documentation in SALAMANDER using the most up-to-date TMAP8, two kinds of warnings and issues occur:

- Alt text is missing for un-captioned media in `docker.md` (this is related to new MOOSE accessibility features recently added to the framework in idaholab/moose#30946).
- The "example cases" link to `examples/index.md` causes an issue when built against certain other MOOSE physics modules, causing a conflict.

## Steps to Reproduce
Build TMAP8 documentation against SALAMANDER's configuration.

## Impact
An annoyance from a documentation perspective, but it is stopping updates/development of TMAP8 related capabilities in SALAMANDER.
